flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -33,6 +37,18 @@ class FlxGamepad implements IFlxDestroyable
 	public var model(default, set):FlxGamepadModel;
 	
 	/**
+	 * For gamepads that can have things plugged into them (the Wii Remote, basically).
+	 * Making the user set this helps Flixel properly interpret inputs properly. 
+	 * EX: if you plug a nunchuk into the Wii Remote, you will get different values for 
+	 * certain buttons than with the Wii Remote alone.
+	 * (This is probably why Wii games ask the player what control scheme they are using.)
+	 * 
+	 * In the future, this could also be used for any attachment that exposes new API features
+	 * to the controller, e.g. a microphone or headset
+	 */
+	public var attachment(default, set):FlxGamepadModelAttachment;
```

Comment:
Sure. But I'm also super interested to know if you have a game controller like that in mind; I'm already thinking about other stuff we could do with attachments -- X360 has the microphone/headset for instance, but not sure if we have any chance of exposing API's for that.